opacclient/opacapp/src/main/assets/bibs/Belin_BSH.json

```diff
@@ -7,8 +7,8 @@
         "baseurl": "http://webopac.winbiap.de/best-sabel"
     },
     "geo": [
-        52.52000659999999,
-        13.404954
+        52.51535,
+        13.41368
     ],
     "information": "http://www.bsb-hochschule.de/Service/Bibliothek/Allgemeine_Informationen_zur_Bibliothek.php",
     "state": "Berlin",
```

Comment:
You are absolutely right. The sad part is that we currently do not have a good way to rename the file, as this would delete all accounts users have set up for this library.